prime-router/metadata/organizations-local.yml

```diff
@@ -30,20 +30,20 @@
             filePath: ./upload
         format: CSV
 
-  - name: pa-phd
-    description: Pennsylvennia PHD
+  - name: pima-az-phd
+    description: Pima County, Arizona PHD
```

Comment:
The diff is really weird, but what happened is I deleted "pa" and add "pima-az-phd" organizations.